xblock/reference/plugins.py

```diff
@@ -171,3 +130,42 @@ def __set__(self, xblock, value):
         We don't support this until we figure out what this means
```

Comment:
I'm sorry, I should have been more clear. I was not intending to ask about python syntax. I was hoping that you could replace the comments about prototypes with a description of exactly what is supported about this field. It looks like you are supporting a read-only file system, and the applications of that are not obvious.